### ISSUE TITLE
False Positive in `_ := range ` cases.

### ISSUE THREAD

**Issue by wfnuser** [kyoh86/scopelint#15]
If we write code like this: 
`for _ = range time.Tick(time.Second * 10) {`

Actually we don't care whatever the key is. But scopelint will fail the check. I think it is a false positive.
One way to solve it is pass the check if the key or value in rage is "_".
`
if typedNode.Key.(*ast.Ident).String() == "_" {break;}
`

One of the complete examples looks like following, and it will fail.

```
package main

import (
  "fmt"
  "time"
)


var	retryTime = 3
var retryItvl = time.Second

func RunBackgroup() {
	go func() {
		for _ = range time.Tick(time.Second * 10) {
    		ReloadIfNeeded()
	    }
	}()
}

func ReloadIfNeeded() {
}

func main() {
	f := func() error {
		var errf error
		return errf
	}

  if err := Retry(f, retryTime, retryItvl); err != nil {
		fmt.Println(err.Error())
		return
  }
  
}

func Retry(f func() error, maxTime int, interval time.Duration) error {
	var errr error
	for tryTime := 0; tryTime < maxTime; tryTime++ {
		if errr = f(); errr == nil {
			return nil
		}
		time.Sleep(interval)
	}
	return errr
}
```

**Comment by kyoh86** [comment]
This is the OBSOLETED project. Please try looppointer or exportloopref instead.

*out of context*

- You can format a code with `gofmt`.
- You can use "code fence" to post a code.
  - https://docs.github.com/en/free-pro-team@latest/github/writing-on-github/creating-and-highlighting-code-blocks#fenced-code-blocks
